osfclient/tests/test_cli.py

```diff
@@ -80,3 +80,13 @@ def simple_config(key):
     expected = ('specify a project ID via the command line, configuration '
                 'file or environment variable')
     assert expected in e.value.args[0]
+
+def test_password_prompt():
+
```

Comment:
Can you add a blank line above `def test_password_prompt` and delete the blank after it? Just to make the `pep8` gods happy :smile: